sdk/tools-windowsservercore-ltsc2016/Dockerfile

```diff
@@ -10,10 +10,10 @@ RUN mkdir $env:ANDROID_HOME; \
     Expand-Archive android_sdk_tools.zip -DestinationPath C:\Android\ ; \
     Remove-Item -Path android_sdk_tools.zip
 
-ENV PATH ${PATH}:${ANDROID_HOME}\\tools:${ANDROID_HOME}\\tools\\bin:${ANDROID_HOME}\\platform-tools
+RUN setx PATH "%PATH%;C:\\Android\\tools;C:\\Android\\tools\\bin;C:\\Android\\platform-tools"
```

Comment:
I think you need to get rid of the double slashes here.  I end up getting confused on this though, which part Dockerfile expects to be escaped and which part Windows needs not escaped.

Reply:
You also may need `[System.Environment]::SetEnvironmentVariable` here, since I'm pretty sure this is Powershell.  https://stackoverflow.com/questions/42092932/appending-to-path-in-a-windows-docker-container

Reply:
Right! Also it should use `;` as a delimiter. Windows! I merged this PR from my phone and didn't see your comment. It didn't fix the issue. Trying another approach...